assets/src/data/eventespresso/schema/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Returns an action object used to update the store with the provided schema
+ * for the provided modelName.
+ *
+ * @param {string} modelName
+ * @param {Object} schema
+ * @return {{type: string, modelName: *, schema}}  The action object.
+ */
+export function receiveSchemaForModel( modelName, schema = {} ) {
+	return {
+		type: 'RECEIVE_SCHEMA_RECORD',
```

Comment:
It seems its pretty much convention in react land to do actions this way.  The `receveSchemaForModel` function is acting like a constant of sorts because it takes care of setting the `type`.  So I don't think making the `RECEIVE_SCHEMA_RECORD` string a constant is of any added value.